models/tmp/stg_iterable__user_history_tmp.sql

```diff
@@ -0,0 +1 @@
+select * from {{ var('user_history') }}
```

Comment:
Super tiny, can you update the format to be like the below:
```sql
select *
from {{ var('table_name') }}
```
Just to keep with consistency across packages.